lib/components/map/default-map.js

```diff
@@ -141,6 +141,13 @@ class DefaultMap extends Component {
 
     return (
       <MapContainer>
+        {/* Add the mapbox wordmark if set in config. The implementing
+          application must include CSS that properly displays the wordmark. See
+          mapbox website for example CSS:
+          https://docs.mapbox.com/help/how-mapbox-works/attribution/#other-mapping-frameworks */}
+        {mapConfig.addMapboxWordmark &&
+          <a href='http://mapbox.com/about/maps' class='mapbox-wordmark' target='_blank'>Mapbox</a>
```

Comment:
It should be possible to use React-Leaflet 's built-in [attribution control](https://react-leaflet.js.org/docs/en/components#attributioncontrol) or Leaflet's equivalent. Doing so would avoid all of the CSS styling proposed in https://github.com/ibi-group/configurations/pull/82. This may require a change in OTP-UI instead of OTP-RR. Thoughts?